templates/egressfirewall.go

```diff
@@ -29,6 +29,12 @@ var EgressFirewallTemplate = ovnv1.EgressFirewall{
 				},
 				Type: ovnv1.EgressFirewallRuleAllow,
 			},
+			{
+				To: ovnv1.EgressFirewallDestination{
+					CIDRSelector: "100.64.0.0/16",
```

Comment:
Any issues if this is added to SDN NetworkPolicy? If there's none, better add it there as well.

Reply:
No, there is not any issue if we add it to SDN as well, I will add it there.